core/utils/mailbox_prom.go

```diff
@@ -38,6 +38,7 @@ func (m *MailboxMonitor) Start(context.Context) error {
 	return m.StartOnce("MailboxMonitor", func() error {
 		t := time.NewTicker(WithJitter(mailboxPromInterval))
 		m.stop = t.Stop
+		// FIXME: monitorLoop never exits because t.Stop does not close the channel
```

Comment:
:facepalm:

Reply:
I see these in some test timeout dumps. Attempting a fix here: https://github.com/smartcontractkit/chainlink/pull/9491